api/envoy/config/core/v3/protocol.proto

```diff
@@ -345,6 +345,22 @@ message Http2ProtocolOptions {
   // <https://www.iana.org/assignments/http2-parameters/http2-parameters.xhtml#settings>`_ for
   // standardized identifiers.
   repeated SettingsParameter custom_settings_parameters = 13;
+
+  // Send HTTP/2 PING frames at this period, in order to test that the connection is still alive.
```

Comment:
Thoughts on making this a dedicated message, with the sub-fields all required?

Reply:
I don't care either way

Reply:
I think I would recommend that if you don't mind either way.

Reply:
Yeah, +1 here.